src/buttons/docs/readme.md

```diff
@@ -1,3 +1,36 @@
-There are two directives that can make a group of buttons behave like a set of checkboxes, radio buttons, or a hybrid where radio buttons can be unchecked.
+With the buttons directive, we can make a group of buttons to behave like a set of checkboxes (`uib-btn-checkbox`) or to behave like a set of radio buttons (`uib-btn-radio`).
 
-In order to be able to uncheck the currently selected radio button, one can use the `uncheckable` attribute in tandem with `uib-btn-radio`.
+### uib-btn-checkbox settings
+
+* `ng-model` -
+  Model where we set the checkbox status. By default `true` or `false`.
+  
+* `btn-checkbox-true`
+  _(Default: `true`)_ -
+  Sets the value for the checked status.
+
+* `btn-checkbox-false`
+  _(Default: `false`)_ -
+  Sets the value for the unchecked status.
+
+### uib-btn-radio settings
+
+* `uib-btn-radio` -
+  Value to assign to the `ng-model` if we check this radio button.
+
+* `ng-model` -
+  Model where we set the radio status. All radio buttons in a group should use the same `ng-model`.
+  
+* `uncheckable`
+  _(Boolean attribute)_ -
```

Comment:
Uhmmm, yeah... But since it is a "boolean" attribute, I felt like it was implicit.